target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -92,6 +92,13 @@ tplink,tl-wr841-v11)
 	ucidef_set_led_switch "lan3" "LAN3" "tp-link:green:lan3" "switch0" "0x04"
 	ucidef_set_led_switch "lan4" "LAN4" "tp-link:green:lan4" "switch0" "0x02"
 	;;
+ubnt,bullet-m|\
+ubnt,nano-m|\
+ubnt,rocket-m)
+	ucidef_set_led_rssi "rssilow" "RSSILOW" "ubnt:red:link1" "wlan0" "1" "100" "0" "13"
```

Comment:
I'm quite sure the last two parameters are not really used on your boards. These are used to change the brightness of pwm leds. According to your dts it looks like the rssi leds are "normal" ones.

Reply:
I'll fix and retest that later today. LEDs are indeed without PWM capability with regards to underlying hardware. Thanks for noticing.